lib/pbench/server/auth/__init__.py

```diff
@@ -126,23 +126,23 @@ def get(
         Returns:
             Response from the request.
         """
-        return self._method("GET", path, None, None, headers=headers, **kwargs)
+        return self._method("GET", path, headers=headers, **kwargs)
 
     def post(
         self,
         path: str,
-        data: Union[Dict, str] = None,
-        json: Optional[Dict] = None,
-        headers: Optional[Dict] = None,
+        data: Optional[Any] = None,
```

Comment:
I think `Optional` is redundant with `Any` because `Any` includes `None`; but whatever. 😆

Reply:
So "anything including none *and* including none"?? Yeah, sure, why not?

Reply:
Literally, "optionally anything or nothing", or colloquially "whatever, I don't care"...

Reply:
So, we really ought to have a definition for that!
```
WHATEVER = Optional[ANY]

...

    def post(
        ...
        data: WHATEVER = None,
        ...
    ) -> requests.Response:
```